src/main/java/bean/Orders.java

```diff
@@ -2,30 +2,31 @@
 
 import javax.persistence.*;
 import java.util.Date;
+import java.util.List;
 
 @Entity
 @Table(name = "Orders")
 public class Orders {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "id")
-    int id;
+    private int id;
     @Column(name = "status_order_id")
-    String statusOrderId;
+    private String statusOrderId;
     @Column(name = "data_order")
-    Date dataOrder;
+    private Date dataOrder;
     @Column(name = "comment_order")
-    String commentOrder;
+    private String commentOrder;
     @Column(name = "total_price")
-    float totalPrice;
+    private float totalPrice;
 
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name="id")
+    @JoinColumn(name = "orders_id")
```

Comment:
Should be something like user_id or another field, which in your db instead of orders_id